modules/server/src/main/ts/bedrock/core/Lifecycle.ts

```diff
@@ -12,6 +12,10 @@ const exitDelay = (driver: Browser, delayExiting: boolean) => {
   return delayExiting ? driver.pause(17 * 60 * 1000) : Promise.resolve();
 };
 
+const markLambdaTest = async (driver: Browser, status: 'failed' | 'passed') => {
+  await driver.executeScript('if (window.lambda_status) { lambda_status=' + status + ' }', []);
```

Comment:
Could we link this to the docs and also add some comment why it's needed. I can't seem to find `lambda_status` all I see is this that uses `lambda-status` didn't look very hard though.

Reply:
Ok could you write some comments in the code here or refer to more information in the ticket. Just want to avoid the next dev being confused about this does in N months from now and then having to search though forums etc to find the thing again.

Reply:
Linked documentation in ticket and added relevant information. Added comments to the code and the ticket reference